doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -449,11 +449,11 @@ See the following instructions for programming the images separately:
 
            west flash --erase
 
-   .. group-tab:: nrfjprog
+   .. group-tab:: nRF Util
 
       1. Open a command prompt in the build folder of the network sample and enter the following command to erase the flash memory of the network core and program the network sample::
 
-           nrfjprog -f NRF53 --coprocessor CP_NETWORK --program zephyr/zephyr.hex --chiperase
+           nrfutil device program --firmware zephyr/zephyr.hex  --options chip_erase_mode=ERASE_ALL --core Network
```

Comment:
Why zephyr/zephyr.hex?
Why not zephyr.hex or build/zephyr/zephyr.hex?
The command as is assumes that users are in the build folder. Would it not be more natural to just say "zephyr.hex", since users can take the file from anywhere?